images/prow-tests/in-gvm-env.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
What's the difference between this script and `my-gvm.sh`, can we give them better names, or add a README to explain what are the scripts supposed to do? And also it seems the code in the two scripts are somewhat duplicated.

Reply:
Added comments to each.

Renaming `my-gvm.sh` => `source-gvm-and-run.sh`